tpenFixtures.json

```diff
@@ -0,0 +1,280 @@
+           ],
+           "target":"https://iiif.io/api/cookbook/recipe/0139-geolocate-canvas-fragment/canvas.json",
+           "viewer":"https://static.t-pen.org/66fd704e021ef957b034eb95/66fd813344b9285591c05abf",
+           "partOf": "66fd826f6d4fd13aaa75ba81"
+        }
+   ],
+   "creator":"62572ba71d974d1311abd673"
+}
+
+// Page
+{
+   "_id":"66fd813344b9285591c05abf",
+   "@type":"Page",
+   "label": "Fixture Page 1",
+   "project":{
+      "id":"66fd704e021ef957b034eb95",
+      "viewer":"https://static.t-pen.org/66fd704e021ef957b034eb95",
+      "tools":[
+         
+      ],
+      "options":{
+         
+      },
+      "license":"CC-BY"
+   },
+   "canvas":"https://iiif.io/api/cookbook/recipe/0139-geolocate-canvas-fragment/canvas.json",
+   "partOf": "66fd826f6d4fd13aaa75ba81",
+   "annotations":[
+      "https://devstore.rerum.io/v1/id/66fd813344b9285591c05abf"
+   ],
+   "viewer":"https://static.t-pen.org/66fd704e021ef957b034eb95/66fd813344b9285591c05abf",
+   "creator":"62572ba71d974d1311abd673",
+}
+
+// Line
+{
+   "_id":"66fd80dc91b50d9f5b1ef5f8",
+   "@type":"Line",
+   "creator":"62572ba71d974d1311abd673",
+   "textualBody":"MAP CONTENT",
+   "project":"66fd704e021ef957b034eb95",
+   "canvas":"https://iiif.io/api/cookbook/recipe/0139-geolocate-canvas-fragment/canvas.json",
+   "layer":"66fd826f6d4fd13aaa75ba81",
+   "viewer":"https://static.t-pen.org/66fd704e021ef957b034eb95/66fd813344b9285591c05abf/66fd80dc91b50d9f5b1ef5f8",
+   "license":"CC-BY"
+}
+
+// Group of 3 Users
+{
+   "_id":"66fd7d0f3d5367e49b2d7bb5",
+   "@type":"Group",
+   "name":"Fixture Group 1",
+   "members":[
+      "66fd78ab3d5367e49b2d7baf",
+      "66fd78ea3d5367e49b2d7bb0",
+      "66fd79213d5367e49b2d7bb1"
+   ]
+}
+
+// 3 Users
+{
+   "_id":"66fd78ab3d5367e49b2d7baf",
+   "@type":"User",
+   "name":"Patrick C.",
+   "email":"patrick@rerum.io",
+   "agent":"https://store.rerum.io/v1/id/771d43bc490d045b8ba04d39",
+   "profile":{
+      "nickname":"Cuba",
+      "github_username":"cubap",
+      "steam_username":"cubap123",
+      "favorite_number":3,
+      "favorite_color":"orange"
+   }
+}
+
+{
+   "_id": "66fd78ea3d5367e49b2d7bb0",
+   "@type":"User",
+   "name":"Bryan H.",
+   "email":"bryan@rerum.io",
+   "agent":"https://store.rerum.io/v1/id/661d43bc490d045b8ba04d38",
+   "profile":{
+      "nickname":"Habes",
+      "github_username":"thehabes",
+      "playstation_username":"thehabes",
+      "favorite_number": 13
+   }
+}
+
+{
+   "_id":"66fd79213d5367e49b2d7bb1",
+   "@type":"User",
+   "name":"Victor O.",
+   "email":"victor@rerum.io",
+   "agent":"https://store.rerum.io/v1/id/66fd78343f9bfbbf656690fc",
+   "profile":{
+      "nickname":"Vic Titan",
+      "github_username":"vyoo",
+      "favorite_color":"purple"
+   }
+}
+
+// Manifest
+{
+   "@context": "http://iiif.io/api/presentation/3/context.json",
+   "@id": "https://devstore.rerum.io/v1/id/66fd818673a1f9339012d26c",
+   "type":"Manifest",
```

Comment:
or `type:"sc:Manifest"`?

Reply:
oo that's a good point.  This is a IIIF Presentation API 3 Manifest which doesn't do the `sc:` bit.  What you are thinking of is a a IIIF Presentation API 2 Manifest and we need one of those.  Maybe I will duplicate this Manifest so we have the same one in both v2 and v3.